include/swift-c/DependencyScan/DependencyScan.h

```diff
@@ -493,9 +482,9 @@ swiftscan_cas_store(swiftscan_cas_t cas, uint8_t *data, unsigned size,
 /// argc and \c argv with \c input. Return \c CacheKey as string.
```

Comment:
The doc-comment still mentions `kind` parameter.
Is the `input` parameter parameter the file path of one of the input source files? If yes, I think the term the swift compiler uses is "primary source file", so a more clear parameter name would be `primary_source_file`.
Also is the `input` parameter supposed to be NULL for a WMO invocation? If so mention it in the doc-comment.